ansible/vars/all.yml

```diff
@@ -2,9 +2,11 @@ manage_subscription: false
 rhel_username:
 rhel_password:
 rhel_pool_id:
+manage_repos: false
 
-prometheus_logging: true
 setup_microshift_host: false
+prometheus_logging: true
+install_microshift: false
```

Comment:
This is true by default for all the cases except RHEL4Edge, right? Should we default to true and update the README.md with one more case?

Reply:
Correct, the defaults I've set are for the RHEL4Edge case. I'll add an examples section to the README.